pkg/api/defaults.go

```diff
@@ -521,7 +602,7 @@ func (p *Properties) setDefaultCerts() (bool, []net.IP, error) {
 
 	var azureProdFQDNs []string
 	for _, location := range helpers.GetAzureLocations() {
-		azureProdFQDNs = append(azureProdFQDNs, FormatAzureProdFQDNByLocation(p.MasterProfile.DNSPrefix, location))
+		azureProdFQDNs = append(azureProdFQDNs, FormatAzureProdFQDNByLocation(p.MasterProfile.DNSPrefix, location, p.GetCustomCloudName()))
```

Comment:
Do we want to append the `p.GetCustomCloudName()` return value when we're not in an Azure Stack context? (e.g., the return value will be `""` in that case)

Reply:
Nevermind I see that addition has been added as the new arg to the `FormatAzureProdFQDNByLocation` func.